index.html

```diff
@@ -14,15 +14,18 @@
         <nav class="nav-list">
             
             <a class="logo" href="#">Leslie Gudo</a>
-            <ul class="navbar-list">
-
-            <li class="menu-icon">
-                <img class="menu" src="static/images/Union.png" alt=" navigation" />
-            </li>
-            <li class="desktop-nav-list">Portfolio</li>
-            <li class="desktop-nav-list">About</li>
-            <li class="desktop-nav-list">Contact</li>
+            <div id="modal" class="modal hidden">
+              <span id="modalClose" class="modalClose">&times;</span>
+            <ul id="navLinks" class="navbar-list">
+            <li class="desktop-nav-list" id="portfolio-1"><a href="#works-1">Portfolio</a></li>
+            <li class="desktop-nav-list" id="about-1"><a href="#about">About</a></li>
+            <li class="desktop-nav-list" id="contact-1"><a href="#contact">Contact</a></li>
           </ul>
+          </div>
+            <a href="#" id="modalOpen" class="menu-icon">
+              <i class="fa fa-bars"></i>
+            </a>
+          
```

Comment:
[OPTIONAL] Well done on the popup menu 🎉 it looks excellent but it would look even better if the close button were in the same place as the hamburger icon so that the switch would be 🍔 ➡️ ❌. The user would then feel like it's not another icon appearing but the same icon that is changing, which would definitely improve your user experience. See what I mean 👇🏽 

## When it's closed
![Screenshot 2023-01-31 at 08 41 14](https://user-images.githubusercontent.com/45739963/215776164-17c55088-38f4-41f6-a963-672791310dd7.png)

## when it's opened
![Screenshot 2023-01-31 at 08 41 51](https://user-images.githubusercontent.com/45739963/215776304-6db2765e-f8d7-45ca-bb62-9e0d5802ba1d.png)

That being said, it looks good but it could look better 😉  keep going 💪🏽 🎊